src/pages/TrueSightV2/components/DisplaySettings.tsx

```diff
@@ -0,0 +1,249 @@
+import { Trans } from '@lingui/macro'
+import { useRef, useState } from 'react'
+import { Sliders } from 'react-feather'
+import { useParams } from 'react-router'
+import { Text } from 'rebass'
+import styled from 'styled-components'
+
+import Column from 'components/Column'
+import Divider from 'components/Divider'
+import ExpandableBox from 'components/ExpandableBox'
+import Icon from 'components/Icons/Icon'
+import Popover from 'components/Popover'
+import { RowBetween, RowFit } from 'components/Row'
+import Toggle from 'components/Toggle'
+import useMixpanel, { MIXPANEL_TYPE } from 'hooks/useMixpanel'
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import useTheme from 'hooks/useTheme'
+import { ApplicationModal } from 'state/application/actions'
+import { useToggleModal } from 'state/application/hooks'
+import { useTokenAnalysisSettings, useUpdateTokenAnalysisSettings } from 'state/user/hooks'
+
+import useKyberAITokenOverview from '../hooks/useKyberAITokenOverview'
+import { HeaderButton } from '../pages/SingleToken'
+import { DiscoverTokenTab } from '../types'
+
+const SettingsWrapper = styled.div`
+  padding: 16px;
+  border-radius: 20px;
+  min-width: 340px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  background-color: ${({ theme }) => theme.tableHeader};
+`
+
+const ViewTutorialButton = styled(RowFit)`
+  color: ${({ theme }) => theme.text};
+  font-size: 12px;
+  font-weight: 500;
+  gap: 2px;
+  cursor: pointer;
+
+  :hover {
+    color: ${({ theme }) => theme.subText};
+  }
+`
+
+const onChainAnalysisSettings = [
+  {
+    id: 'numberOfTrades',
+    name: 'Number of Trades / Type of Trade',

```

Comment:
```suggestion
    name: t`Number of Trades / Type of Trade`,
```
use this instead of `<Trans>{t.name}</Trans>`